packages/fx-core/src/common/arm.ts

```diff
@@ -13,10 +13,10 @@ export interface BicepOrchestrationTemplate {
 }
 
 export interface BicepOrchestrationParameterTemplate extends BicepOrchestrationTemplate {
-  ParameterFile: string;
+  ParameterFile?: string;
 }
 
-export interface BicepOrchestrationResourceTemplate extends BicepOrchestrationTemplate {
+export interface BicepOrchestrationModuleTemplate extends BicepOrchestrationTemplate {
   Outputs: { [OutputName: string]: string };
```

Comment:
Thanks for making some properties optional. The output here may also be optional?

Reply:
Good catch. Fixed.